apps/pie-microsite/src/_11ty/shortcodes/tokensTable/index.js

```diff
@@ -178,6 +190,30 @@ const buildTokensListForCategory = (tokens, path, category, tokenType) => {
     return buildTokensList(tokenListElements);
 };
 
+/**
+ * Builds a categorised list of tokens
```

Comment:
This comment seems at odds with the function name?

The function is called `buildUncategorisedLists`, but the comment says it builds a ategorised list?

Reply:
ah yep, sorry copy and paste issue! will fix 👍